lib/main.js

```diff
@@ -205,6 +208,7 @@ exports.main = function(options, callbacks) {
       }
     }
     noatb.install();
+    oninstall.install(is_install);
```

Comment:
Will this be called every time the extension loads? Is there any benefit to only calling it if `is_install` is true?